style.css

```diff
@@ -89,21 +110,47 @@ header {
   display: flex;
   justify-content: space-between;
   align-items: center;
+  min-height: 30px;
+  padding: 0 24px;
+}
+
+.navigation-two {
+  display: flex;
+  gap: 50px;
+  margin-right: 70px;
+  z-index: 10;
+  text-decoration: none;
+}
+
+.nav {
+  transition: 0.7s ease;
+  z-index: 10;
+}
+
+.nav a {
+  text-decoration: none;
+  color: #fff;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
 }
```

Comment:
- [ ] Great job working on the styles for the links! 💯 However, you changed the color of the text for the mobile version, and this affected the desktop version, as shown here:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/202288265-be1dca17-a284-4931-bd1b-db0754e08568.png) | ![imageedit_2_3553203714](https://user-images.githubusercontent.com/77069011/202288316-4324239d-b9da-4a33-bf47-5e7b06fe537e.gif) |

Please, consider fixing this! 🤓 🛠️